packages/gatsby/src/schema/__tests__/run-query.js

```diff
@@ -275,752 +307,1171 @@ it(`should use the cache argument`, async () => {
-        result.forEach(edge => {
-          expect(edge.string).not.toEqual(`b`)
-          expect(edge.string).not.toEqual(`c`)
+          expect(result.length).toEqual(1)
+          result.forEach(node => {
+            expect(node.string).not.toEqual(`b`)
+            expect(node.string).not.toEqual(`c`)
+          })
         })
-      })
 
-      it(`handles the nin operator for ints`, async () => {
-        let result = await runFilter({ index: { nin: [0, 2] } })
+        it(`handles the nin operator for ints`, async () => {
+          const [result] = await runFilter({ index: { nin: [0, 2] } })
 
-        expect(result.length).toEqual(1)
-        result.forEach(edge => {
-          expect(edge.index).not.toEqual(0)
-          expect(edge.index).not.toEqual(2)
+          expect(result.length).toEqual(1)
+          result.forEach(node => {
+            expect(node.index).not.toEqual(0)
+            expect(node.index).not.toEqual(2)
+          })
         })
-      })
 
-      it(`handles the nin operator for floats`, async () => {
-        let result = await runFilter({ float: { nin: [1.5] } })
+        it(`handles the nin operator for floats`, async () => {
+          const [result] = await runFilter({ float: { nin: [1.5] } })
 
-        expect(result.length).toEqual(2)
-        result.forEach(edge => {
-          // Might not have the property (-> undefined), must not be 1.5
-          expect(edge.float).not.toEqual(1.5)
+          expect(result.length).toEqual(2)
+          result.forEach(node => {
+            // Might not have the property (-> undefined), must not be 1.5
+            expect(node.float).not.toEqual(1.5)
+          })
         })
-      })
 
-      it(`handles the nin operator for booleans`, async () => {
-        let result = await runFilter({ boolean: { nin: [true, null] } })
+        it(`handles the nin operator for booleans`, async () => {
+          const [result] = await runFilter({
+            boolean: { nin: [true, null] },
+          })
 
-        // Do not return the node that does not have the field because of `null`
-        expect(result.length).toEqual(1)
-        result.forEach(edge => {
-          // Must have the property, must not be true nor null
-          expect(edge.boolean !== undefined).toBe(true)
-          expect(edge.boolean !== true && edge.boolean !== null).toBe(true)
+          // Do not return the node that does not have the field because of `null`
+          expect(result.length).toEqual(1)
+          result.forEach(node => {
+            // Must have the property, must not be true nor null
+            expect(node.boolean !== undefined).toBe(true)
+            expect(node.boolean !== true && node.boolean !== null).toBe(true)
+          })
         })
-      })
 
-      it(`handles the nin operator for double null`, async () => {
-        if (IS_LOKI) return
+        it(`handles the nin operator for double null`, async () => {
+          if (IS_LOKI) return
```

Comment:
This will make jest compain, do `expect(true).toBe(true)` and then return. Same for other cases like that here.

Reply:
Can you elaborate in which situation jest is complaining? Tests are passing and local `yarn jest:loki` is not complaining either.

I'm fine with adding the dud pass, especially since we'll rip it soon anyways. Just curious which workflow this is breaking for future reference.

Reply:
It usually says smth like "no assertions found it tests". At least at some point before that was a failure state, it might not be anymore.